### PR TITLE
fix(network): parse value to boolean

### PR DESCRIPTION
**Pull request by angao**
**What type of PR is this?**
/kind bug
/area network

**What this PR does / why we need it**:
- 修改字符串不为空时被当成 boolean true 类型

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @WangMinjie114 @hanxueluo 

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: inventory/deploy-cluster/group_vars/k8s-cluster/k8s-cluster.yml [hunk 1]
```diff
@@ -348,10 +348,6 @@ kube_apiserver_bind_address: "0.0.0.0"
 kube_controller_manager_bind_address: "{{ ip | default(fallback_ips[inventory_hostname]) }}"
 kube_scheduler_bind_address: "{{ ip | default(fallback_ips[inventory_hostname]) }}"
 
-# fixedip-ipam server config
-fixedip_ipam_enabled: false
```
Comment (hanxueluo) [inline inventory/deploy-cluster/group_vars/k8s-cluster/k8s-cluster.yml:352]: 默认值也删了吗？

Comment (angao) [reply inventory/deploy-cluster/group_vars/k8s-cluster/k8s-cluster.yml:352]: 这个没用到，现在变量名是 `fixedip_enabled`

**Comment by hanxueluo** [comment]
LGTM

/assign @WangMinjie114
确认下语法

**Comment by angao** [comment]
经测试，新添加节点无问题

**Pull request merged**